frontend/pages/welcome.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect } from "react";
+import { useRouter } from "next/router";
+import { adminFetchIt } from "../utils";
+import { useAuth, withLoginRequired, withAuth } from "use-auth0-hooks";
+import { AUTH0_API_IDENTIFIER } from "../utils";
+
+const Welcome = () => {
+  const { push } = useRouter();
+  const { accessToken } = useAuth({ audience: AUTH0_API_IDENTIFIER });
+
+  useEffect(() => {
+    const checkUsersRegistry = async () => {
+      try {
+        const data = await adminFetchIt("/registry/admin", accessToken);
+        if (!data) {
```

Comment:
don't think you need the data check here?

the fetchIt should throw an error if nothing was found